avm/ptn/network/hub-networking/main.bicep

```diff
@@ -263,6 +263,7 @@ output hubAzureFirewalls object[] = [
         location: hubAzureFirewall[index].outputs.location
         name: hubAzureFirewall[index].outputs.name
         resourceId: hubAzureFirewall[index].outputs.resourceId
+        privateIp: hubAzureFirewall[index].outputs.privateIp
```

Comment:
Please update minor version in version.json. Although small and backward compatible, this qualifies as a change to the API

Reply:
Updated.